package/com.unity.formats.usd/Runtime/Scripts/IO/Skel/SkeletonImporter.cs

```diff
@@ -160,7 +169,7 @@ public static class SkeletonImporter {
       }
 
       var mesh = smr.sharedMesh;
-      var geomXf = meshBinding.geomBindTransform.value;
+      Matrix4x4 geomXf = UnityTypeConverter.FromMatrix(skinningQuery.GetGeomBindTransform());
```

Comment:
Does the time argument matter here? I *think* the bind pose can be animated, but maybe we don't support that?